tests/unit/EntityMenu.spec.ts

```diff
@@ -18,6 +18,9 @@ const businessStore = useBusinessStore()
 const configurationStore = useConfigurationStore()
 const rootStore = useRootStore()
 
+/** To prevent - Unable to locate target [data-app] */
+console.warn = jest.fn()
```

Comment:
I think there's a better way to prevent this warning. Search the other tests for "data-app".

The problem with mocking console.warn or console.log is that we won't see the warnings or logs from the running code, either (when the test is run), and those are helpful for debugging.